src/planning/task_planner/scripts/task_planner.py

```diff
@@ -101,13 +105,10 @@ def __init__(self):
             'do_nothing' : [LocationOfInterest.LIVING_ROOM, ObjectOfInterest.NONE],
             'fetch_banana' : [LocationOfInterest.LIVING_ROOM, ObjectOfInterest.BANANA],
         }
-
+        self.navigationGoal = None
```

Comment:
navigation goal to be made more modular and easily updatable since there will be several rooms in the facility